telestion-api/build.gradle.kts

```diff
@@ -0,0 +1,30 @@
+plugins {
+    id("telestion-java-library")
+}
+
+// some meta information (important for packaging metadata)
+telestion {
+    prettyName.set("Telestion API")
+    description.set("The essential components for a Telestion Project Application")
+}
+
+tasks.withType<Test>().configureEach {
+    systemProperty("junit.jupiter.displayname.generator.default", "de.wuespace.telestion.api.CamelCase")
+}
+
```

Comment:
What does this do?

Reply:
JUnit uses the method names for test cases by default. To make the test names more readable, we override the test name generator. (e.g. `shouldCallExtendedHandlerOnValidBuffer` maps to `should Call Extended Handler On Valid Buffer`)